content/modules/redisearch/release-notes/redisearch-2.0-release-notes.md

```diff
@@ -5,6 +5,26 @@ weight: 95
 alwaysopen: false
 categories: ["Modules"]
 ---
+## RediSearch 2.0.5 (December, 2020)
+
+This is a maintenance release for version 2.0.
+
+Details:
+
+- Minor features:
+    - [#1696](https://github.com/RediSearch/RediSearch/pull/1696) Add [`MAXAGGREGATERESULTS`](https://oss.redislabs.com/redisearch/Configuring/#maxaggregateresults) module configuration for [`FT.AGGREGATE`](https://oss.redislabs.com/redisearch/Commands/#ftaggregate). Similar to MAXSE`ARCHRESULTS for `FT.SEARCH`, it limits the maximum number of results returned.
```

Comment:
Similar to MAXSE`ARCHRESULTS for

There is a typo here. We probably wanted this:

    Similar to ` MAXSEARCHRESULTS` for